release-notes/major11/11.3.0/valtimo-backend-libraries.md

```diff
@@ -10,7 +10,11 @@ The following features were added:
 
 ## Bugfixes
 
-No bugfixes.
+The following bugs were fixed:
+
+* **Besluit creation fails because of missing non-mandatory property**
+
+  When `reason for expiry` is not selected while configuring 'Create Zaakbesluit' plugin action in the 'Besluiten API' plug-in, the following error occurs and the besluit creation fails: Cannot force empty String ("") to ` com.ritense.besluitenapi.client.Validation`. It's now fixed.
```

Comment:
```suggestion
  When `reason for expiry` is not selected while configuring 'Create Zaakbesluit' plugin action in the 'Besluiten API' plug-in, the following error occurs and the besluit creation fails: Cannot force empty String ("") to ` com.ritense.besluitenapi.client.Validation`.
```
This is in the bugfixes section, so if a bug is mentioned here, it only makes sense it's fixed. So there's no need to write that part down.